modin/experimental/xgboost/xgboost_ray.py

```diff
@@ -648,29 +547,54 @@ def _train(
         return result
 
 
+@ray.remote
+def _map_predict(booster, part, columns, **kwargs):
+    """
+    Run prediction on remote worker.
+
+    Parameters
+    ----------
+    booster : xgboost.Booster or ray.ObjectRef
+        A trained booster.
+    part : pandas.DataFrame or ray.ObjectRef
+        Partition of full data used for local prediction.
+    columns : list or ray.ObjectRef
+        Columns for the result.
+    **kwargs : dict
+        Other parameters are the same as `xgboost.Booster.predict`.
+
+    Returns
+    -------
+    ray.ObjectRef
+        ``ray.ObjectRef`` with partial prediction.
+    """
+    dmatrix = xgb.DMatrix(part)
+    prediction = pandas.DataFrame(
+        booster.predict(dmatrix, **kwargs),
+        index=part.index,
+        columns=columns,
+    )
+    return prediction
+
+
 def _predict(
     booster,
     data,
-    num_actors,
     **kwargs,
 ):
     """
     Run distributed prediction with a trained booster on Ray backend.
 
-    During work it evenly distributes `data` between workers,
-    runs xgb.predict on each worker for subset of `data` and creates
-    Modin DataFrame with prediction results.
+    During work it runs xgb.predict on each worker for row partition of `data`
```

Comment:
```suggestion
    During execution it runs `xgb.predict` on each worker for subset of `data`
```

Reply:
Used the next
```python
During execution it runs ``xgb.predict`` on each worker for subset of `data`
```
because we use one backtick for parameters highlighting, double for code entities